main.nf

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env nextflow
+
+
+// check parameters
+if (!params.inputDir) {
+    exit 1, "inputDir parameter is missing."
+}
+if (!params.outputDir) {
+    exit 1, "outputDir parameter is missing."
+}
+if ( !new File(params.inputDir).exists()) {
+    exit 1, "The input folder does not exists."+params.inputDir+"\n"
+}
+if (!params.outPrefix) {
+    exit 1, "Output prefix parameter is missing."
+}
+if (!params.debug) {
+    exit 1, "Debug prefix parameter is missing."
+}
+if (!params.localOutDir) {
+    params.localOutDir='bam'
+}
+
+pairFiles_ch = Channel.fromFilePairs( params.inputDir+"/*{1,2}.fastq.gz", size: 2, checkIfExists: true )
+
+process ERValign {
+    // tag ${sample}
+    
+    // executor configuration
+    time '8h'
+    // memory '35.GB'
+    scratch true
+    storeDir params.outputDir
+
+    // other configuration
+    echo true
+    errorStrategy 'terminate'
+
+    input:
+    val(outPrefix) from params.outPrefix
+    val(localOutDir) from params.localOutDir
```

Comment:
Need to call localOutDir parameter in nextflow.config file

Reply:
Thanks, forgot to update the template